tests/scripts/all.sh

```diff
@@ -1149,6 +1161,9 @@ component_test_asan_remove_peer_certificate () {
 
     msg "test: compat.sh, !MBEDTLS_SSL_KEEP_PEER_CERTIFICATE"
     if_build_succeeded tests/compat.sh
+
+    msg "test: context-info.sh, !MBEDTLS_SSL_KEEP_PEER_CERTIFICATE"
+    if_build_succeeded tests/context-inf.sh
```

Comment:
This command will not even run. Please make sure that every test you add is manually run at least once.

Reply:
Why do you think? This program should work independently from MBEDTLS SSL_KEEP_PEER CERTIFICATE. That's why I added it here

Reply:
```suggestion
    if_build_succeeded tests/context-info.sh
```

Reply:
Nice check :)